pycqed/measurement/waveform_control/fluxpulse_predistortion.py

```diff
@@ -133,7 +133,7 @@ def convert_expmod_to_IIR(expmod, dt, inverse_IIR=True):
         b = [i[1] for i in iir]
     else:
         A, B, tau = expmod
-        if isinstance(tau, list):  # sum of exp mod
+        if np.array(tau).ndim > 0:  # sum of exp mod
```

Comment:
Maybe convert tau to np.array()?
```
tau = np.array(tau)
if tau.ndim > 0:
```

Reply:
Let's keep it as it is since tau is used as a scalar in the else branch.